code/modules/research/xenobiology/crossbreeding/burning.dm

```diff
@@ -0,0 +1,426 @@
+/*
+Burning extracts:
+	Have a unique, primarily offensive effect when
+	filled with 10u plasma and activated in-hand.
+*/
+/obj/item/slimecross/burning
+	name = "burning extract"
+	desc = "It's boiling over with barely-contained energy."
+	effect = "burning"
+	container_type = INJECTABLE | DRAWABLE
+	icon_state = "burning"
+
+/obj/item/slimecross/burning/Initialize()
+	..()
+	create_reagents(10)
+
+/obj/item/slimecross/burning/attack_self(mob/user)
+	if(!reagents.has_reagent("plasma",10))
+		to_chat(user, "<span class='warning'>This extract needs to be full of plasma to activate!</span>")
+		return
+	reagents.remove_reagent("plasma",10)
+	to_chat(user, "<span class='notice'>You squeeze the extract, and it absorbs the plasma!</span>")
+	playsound(src, 'sound/effects/bubbles.ogg', 50, 1)
+	playsound(src, 'sound/magic/fireball.ogg', 50, 1)
+	do_effect(user)
+
+/obj/item/slimecross/burning/proc/do_effect(mob/user) //If, for whatever reason, you don't want to delete the extract, don't do ..()
+	qdel(src)
+	return
+
+/obj/item/slimecross/burning/grey
+	colour = "grey"
+
+/obj/item/slimecross/burning/grey/do_effect(mob/user)
+	var/mob/living/simple_animal/slime/S = new(get_turf(user),"grey")
+	S.visible_message("<span class='danger'>A baby slime emerges from [src], and it nuzzles [user] before burbling hungrily!</span>")
+	S.Friends[user] = 20 //Gas, gas, gas
+	S.bodytemperature = T0C + 400 //We gonna step on the gas.
+	S.nutrition = S.get_hunger_nutrition() //Tonight, we fight!
+	..()
+
+/obj/item/slimecross/burning/orange
+	colour = "orange"
+
+/obj/item/slimecross/burning/orange/do_effect(mob/user)
+	user.visible_message("<span class='danger'>[src] boils over with a caustic gas!</span>")
+	var/datum/reagents/R = new/datum/reagents(100)
+	R.my_atom = get_turf(user)
```

Comment:
That doesn't make much sense and chem smoke effect copies the values to internal holder to remove this line please.